File: Solutions/Java/1684.java
class Solution {
    public int countConsistentStrings(String allowed, String[] words) {
        int ans = words.length;
        Set<Character> hash_set = new HashSet<>();
        for (int k=0; k<allowed.length(); k++){
            hash_set.add(allowed.charAt(k));
        }
        for(String s : words){
            for(int i=0; i<s.length(); i++){
                if(!hash_set.contains(s.charAt(i))){
                    ans--;
                    break;
                }
            }
        }
        return ans;
    }
}

File: Solutions/Python/1684.py
class Solution:
    def countConsistentStrings(self, allowed: str, words: List[str]) -> int:
        allowed = set(allowed)
        count = 0
        for i in words:
            for j in i:
                if j not in allowed:
                    count += 1
                    break
        return len(words) - count
